mapping-for-navigation/_posts/0000-01-10-circular-junctions.md

```diff
@@ -0,0 +1,208 @@
+
+![][bceeea1b]
+
+  [bceeea1b]: ../../images/intersection-modeling/roundabouts/roundabout_traffic_circle.png "Traffic circle"
+
+The size of circular junctions and rotaries can vary from small to very large with several lanes, however generally **<u>the rotary or circular junction should not have any road network inside itself</u>**, this means circular ring roads around a city are not valid circular junctions.
+
+To designate such case of circular junction use tag **[junction][f14df547]**=**[circular][d11d94d4]**.
+
+  [d11d94d4]: https://wiki.openstreetmap.org/wiki/Tag:junction%3Dcircular
+
+**How to map**
+
+To add such a junction, use the same approach as for drawing roundabouts. Draw a circle line and tag it accordingly to the class of the roads that join it. Mark the appropriate nodes with traffic lights and traffic signs that control the priority of the movement of traffic flows.
+
+If across a traffic circle or a roundabout go some routes that use different parts of that circles you may split the closed line into segments with the similar tagging and add each segment into respective route relation.
+
+[![][8d6d5862]][8d6d5862]
+<sup>_Place Charles de Gaulle is a large road junction in Paris, France, the meeting point of twelve straight avenues_</sup>
+
+  [8d6d5862]: ../../images/intersection-modeling/roundabouts/roundabout_traffic_circle_paris.png "Place Charles de Gaulle is a large road junction in Paris, France, the meeting point of twelve straight avenues"
+
+## Other features that similar to a roundabout but are not so
+
+### Hamburger roundabout/throughabout/cut-through
+
+In spite of their name, [Hamburger roundabouts](https://wiki.openstreetmap.org/wiki/Hamburger_roundabout), also known as [throughabout or cut-through](http://en.wikipedia.org/wiki/Roundabout#Hamburger_roundabout), are not roundabouts. In this situation the tag **[junction][f14df547]**=**[roundabout](https://wiki.openstreetmap.org/wiki/Tag:junction%3Droundabout)** has to be avoided.
+
+|[![][2ff2d8eb]][2ff2d8eb]|[![][b421c5e4]][b421c5e4]|
+|---|---|
+|<sup>_A throughabout divided by double carriageway_</sup>|<sup>_A throughabout divided by bi-directional way_</sup>|
+
+  [2ff2d8eb]: ../../images/intersection-modeling/roundabouts/roundabout_throughabout_kyiv.png
+  [b421c5e4]: ../../images/intersection-modeling/roundabouts/roundabout_throughabout.png
+
+### Turning circle
+
+Turning circle - this is a circular feature with a traversable center point, which allows vehicles to turn around - **[highway][1108080d]**=**[turning_circle][496a5a09]**
+
+[![][86003e49]][86003e49]
+
+  [86003e49]: ../../images/intersection-modeling/roundabouts/roundabout_turning_circle.png
+
+### Turning loop
+
+Turning loop has the same mission as Turning circle but with none-traversable center point.
```

Comment:
> Turning loops are similar to turning circles but with a non-traversable center point.

Reply:
@karenzshea Thank you for your review and fix the wording. But one question: why this sentence in plural? What if we use singular here and above

> Turning loop is similar to the turning circle but with a non-traversable center point.